core/templates/dev/head/pages/exploration_editor/feedback_tab/feedback_tab.html

```diff
@@ -37,8 +37,8 @@ <h2>Exploration Feedback</h2>
             <i class="material-icons oppia-vcenter" title="Return to list of feedback threads">&#xE5CB;</i>
           </button>
         </span>
-        <span style="font-size: larger" ng-show="!activeThread.isSuggestionThread()">Feedback Thread: <[activeThread.subject]></span>
-        <span style="font-size: larger" ng-show="activeThread.isSuggestionThread()">Suggestion Thread: <[activeThread.subject]></span>
+        <span style="font-size: larger" ng-show="!activeThread.isSuggestionThread() && !activeThread.suggestion">Feedback Thread: <[activeThread.subject]></span>
```

Comment:
Add TODO here and below if you plan to remove this after switching to the new framework. Otherwise we'll forget.

Better still, in these places, just do an ng-if for "new/old framework", so that when you do a sweep later to find all instances of the old framework, you'll find them all, including the occurrences here. (If you do that you don't need the TODO.)

Reply:
Done... Did the second thing you suggested.